src/main/java/uk/gov/companieshouse/officerfiling/api/utils/LogHelper.java

```diff
@@ -3,32 +3,136 @@
 import java.util.HashMap;
 import java.util.Map;
 import org.springframework.stereotype.Component;
+import uk.gov.companieshouse.api.model.transaction.Transaction;
+
+import javax.servlet.http.HttpServletRequest;
 
 @Component
 public final class LogHelper {
 
-    private LogHelper() {
-        // intentionally blank
+    private enum Key {
+        MESSAGE,
+        START,
+        END,
+        DURATION,
+        STATUS,
+        METHOD,
+        PATH,
+        OFFSET,
+        IDENTITY,
+        TRANSACTION_ID,
+        FILING_ID,
+        COMPANY_NUMBER,
+        COMPANY_NAME;
```

Comment:
Some of these keys are defined in `LogContextProperties` in the structured logging library. Can the existing keys be used for consistency please?